src/global_vars.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+//ignoring typical include guard in favor of pragma
+
+extern int memorized_tile_count;
```

Comment:
This would probably fit better in the existing `cached_options.h` rather than making a new header (and then you wouldn't have to battle `clang-tidy` about header guards).

Reply:
Mm. Fair, but now I'm including `cached_options.h` in `avatar.cpp` where it might not belong.